fingerprint.go

```diff
@@ -23,19 +33,34 @@ func NewFingerprint(b []byte) *Fingerprint {
 }
 
 // FingerprintFile is used to generate a fingerprint from a
-// file stored on a disk. The parameter to the function must be a path
+// file stored on a disk. The path parameter must be a path
 // to a valid file in supported format.
 func (x *Client) FingerprintFile(path string) (*Fingerprint, error) {
-	return newFingerprint([]byte(path), true)
+	return x.FingerprintFileForTypes(path, FingerprintTypeAll)
+}
+
+// FingerprintFileForTypes is used to generate a fingerprint from a
+// file stored on a disk. The path parameter must be a path to a
+// valid file in supported format. The typ parameter specifies
+// which types of fingerprints to create.
+func (x *Client) FingerprintFileForTypes(path string, typ FingerprintType) (*Fingerprint, error) {
```

Comment:
Just something to consider. We don't really need new function for this, we can define the `FingerprintFile()` function as `FingerprintFile(path string, types ...FingerprintType)` that way you can call it like:

   FingerprintFile(path)

but also

    FingerprintFile(path, FingerprintTypeAudio, FingerprintTypeVideo)

Reply:
I never liked this syntax in Go or C cause it allowed you to provided multiple parameters when you should only be providing one.

Reply:
Actually, I changed my mind and made it the way you suggested.